main/CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ if(WIN32)
   set_property(TARGET rootcling APPEND_STRING PROPERTY LINK_FLAGS " -STACK:4000000")
 endif()
 
+ROOT_EXECUTABLE(rootbrowse src/rootbrowse.cxx LIBRARIES RIO Core Rint Gui)
```

Comment:
```suggestion
if(NOT minimal)
   ROOT_EXECUTABLE(rootbrowse src/rootbrowse.cxx LIBRARIES RIO Core Rint Gui)
endif()
```

Reply:
I'd rather not add complexity to the build system if not necessary - in minimal `rootbrowse` will be compiled and just fail gracefully to open.

Reply:
Sure, if that gets merged I'll take care of it - should be hard to miss, since I always use minimal in one of my build trees :)